internal/sidekick/api/model.go

```diff
@@ -747,8 +750,6 @@ type Field struct {
 	// - For OpenAPI, it is an optional field
 	// - For OpenAPI, it has format == "uuid"
 	AutoPopulated bool
-	// IsResourceReference is true if the field is annotated with google.api.resource_reference.
-	IsResourceReference bool
```

Comment:
This looks redundant now.  If we are going to add `ResourceReference` then we should add a helper:

```
func (f *field) IsResourceReference() bool {
    return f.ResourceReference != nil
}
```
-- @coryan